examples/ny-tlc-report-external-dependencies.yaml

```diff
@@ -30,11 +30,13 @@ spec:
       persistentVolumeClaim:
         claimName: pvc-ksv
   driver:
-    volumeMounts:
-      - name: job-deps
-        mountPath: /dependencies
+    config:
+      volumeMounts:
+        - name: job-deps
+          mountPath: /dependencies
   executor:
-    instances: 3
-    volumeMounts:
-      - name: job-deps
-        mountPath: /dependencies
+    config:
+      replicas: 3
```

Comment:
Replicas should be at same level as config.

Reply:
We agreed that `replicas` is outside of `config`.